sqlglot/dialects/tsql.py

```diff
@@ -200,6 +200,17 @@ def inner_func(args: t.List) -> E:
     return inner_func
 
 
+def setitem_sql(self, expression: exp.SetItem) -> str:
+    """tsql does not use '=' in SET command but does use '=' with Variables"""
```

Comment:
what does this comment mean?

Reply:
Documenting differences in SET syntax within TSQL. Comment is now gone as code incorporated into Generator.